e2e/_suites/fleet/stand-alone.go

```diff
@@ -102,11 +85,6 @@ func (sats *StandAloneTestSuite) startAgent(image string, env map[string]string)
 		dockerImageTag += "-amd64"
 	}
 
-	configurationFilePath, err := steps.FetchBeatConfiguration(true, "elastic-agent", "elastic-agent.docker.yml")
```

Comment:
I think if we convert the file int a static one, instead of dynamic, we will lose the ability to use the elastic-agent configuration file from a PR, in the case it's modified.

Reply:
I rollback my previous commit with #981

Reply:
Can we check other usages off the FetchBeatConfiguration method? If it's not needed in the metricbeat suite, then we could completely remove it